web/tests/common.test.js

```diff
@@ -44,6 +44,8 @@ run_test("adjust_mac_kbd_tags mac", ({override}) => {
         ["Enter", "Return"],
         ["Ctrl", "⌘"],
         ["Alt", "⌥"],
+        ["Ctrl-[", "Cmd-["],
+        ["Ctrl+K", "⌘+K"],
         ["#stream_name", "#stream_name"],
         ["Ctrl+K", "Ctrl+K"],
```

Comment:
This test passes, but is broken with your changes, as this line undoes the test item created by your change above.